frontend/src/scenes/saved-insights/savedInsightsLogic.ts

```diff
@@ -28,8 +28,10 @@ export interface SavedInsightFilters {
     search: string
     insightType: string
     createdBy: number | 'All users'
-    dateFrom?: string | Dayjs | undefined
-    dateTo?: string | Dayjs | undefined
+    dateFrom: string | Dayjs | undefined | 'all'
+    dateTo: string | Dayjs | undefined
+    limit: number
```

Comment:
IMO better to have `limit` hardcoded as there's no feature for changing page size in the UI. Then the URL could use human-friendlier `page` rather than `offset` that exposes DB-related implementation details.

Reply:
Sure